packages/yoastseo/src/app.js

```diff
@@ -337,12 +335,9 @@ App.prototype.changeAssessorOptions = function( assessorOptions ) {
  * @returns {Assessor} The assessor instance.
  */
 App.prototype.getSeoAssessor = function() {
-	const { useCornerStone, useKeywordDistribution } = this._assessorOptions;
```

Comment:
Similarly, there is still another reference to `useKeywordDistribution` in this file, see line 288.